ML 0.3  Real life application of Data Mining (A) #7643

Ruchika-20 opened the pull request:
<hr>

## Description 📜

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. 
Fixes #issue_number_here 

<hr>

## Type of change 📝

<!----Please delete the hashtag from the correct option----->

- [x] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [ ] Doc#umentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

<!----Please delete the hashtag from your domain----->

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] JavaScript_#Domain
- [ ] ME#RN
- [x] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

<!----Please delete options that are not relevant.And in order to tick the check box just but x inside them for example [x] like this----->

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

<!----Please delete options that are not relevant.----->

## Screenshots / Gif (Optional) 📸

<hr>

abhisheks008 reviewed (changes requested):
Follow the template and procedure mentioned in the Demo folder, for providing the URLs of your Google Drive link.

Check here, https://github.com/girlscript/winter-of-contributing/tree/Machine_Learning/Machine_Learning/Demo

@Ruchika-20

Ruchika-20 commented:
actually i did not understood the nomenclature of url and asked about the
same to isha but she was not able to see  my msg so i renamed according to
mine understanding.....Can you please correct me so that i can make the
necessary changes?

Thank you.

On Wed, Nov 10, 2021 at 10:02 PM Abhishek Sharma ***@***.***>
wrote:

> Assigned #7706
> <https://github.com/girlscript/winter-of-contributing/pull/7706> to
> @Ruchika-20 <https://github.com/Ruchika-20>.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/girlscript/winter-of-contributing/pull/7706#event-5598417066>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUEIB54KAFJ4XHEOEHJGOITULKNCLANCNFSM5HYINP4A>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>

abhisheks008 commented:
@Ruchika-20 Have you gone through the whole Demo folder?

Ruchika-20 commented:
Yes

On Thu, Nov 11, 2021, 9:21 AM Abhishek Sharma ***@***.***>
wrote:

> @Ruchika-20 <https://github.com/Ruchika-20> Have you gone through the
> whole Demo folder?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/girlscript/winter-of-contributing/pull/7706#issuecomment-965964355>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUEIB55EALT5U3GGLZXHTUDULM4VLANCNFSM5HYINP4A>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>

Isha307 commented:
> actually i did not understood the nomenclature of url and asked about the same to isha but she was not able to see my msg so i renamed according to mine understanding.....Can you please correct me so that i can make the necessary changes? Thank you.
> […](#)
> On Wed, Nov 10, 2021 at 10:02 PM Abhishek Sharma ***@***.***> wrote: Assigned #7706 <#7706> to @Ruchika-20 <https://github.com/Ruchika-20>. — You are receiving this because you were assigned. Reply to this email directly, view it on GitHub <[#7706 (comment)](https://github.com/girlscript/winter-of-contributing/pull/7706#event-5598417066)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AUEIB54KAFJ4XHEOEHJGOITULKNCLANCNFSM5HYINP4A> . Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

@Ruchika-20 

> actually i did not understood the nomenclature of url and asked about the same to isha but she was not able to see my msg so i renamed according to mine understanding.....Can you please correct me so that i can make the necessary changes? Thank you.
> […](#)
> On Wed, Nov 10, 2021 at 10:02 PM Abhishek Sharma ***@***.***> wrote: Assigned #7706 <#7706> to @Ruchika-20 <https://github.com/Ruchika-20>. — You are receiving this because you were assigned. Reply to this email directly, view it on GitHub <[#7706 (comment)](https://github.com/girlscript/winter-of-contributing/pull/7706#event-5598417066)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AUEIB54KAFJ4XHEOEHJGOITULKNCLANCNFSM5HYINP4A> . Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

@Ruchika-20 Sorry I was busy, 
You have to make a markdown file and follow this template "https://github.com/girlscript/winter-of-contributing/blob/Machine_Learning/Machine_Learning/Demo/ML_%23_%23_Audio%20Template_(A).md"

Ruchika-20 commented:
[![Your_Title](Image_link_for_Audio_thumbnail)](Your_audio_link "Your_title")

please explain this format


On Thu, Nov 11, 2021 at 5:53 PM isha ***@***.***> wrote:

> actually i did not understood the nomenclature of url and asked about the
> same to isha but she was not able to see my msg so i renamed according to
> mine understanding.....Can you please correct me so that i can make the
> necessary changes? Thank you.
> … <#m_5261266020893853437_>
> On Wed, Nov 10, 2021 at 10:02 PM Abhishek Sharma *@*.***> wrote: Assigned
> #7706 <https://github.com/girlscript/winter-of-contributing/pull/7706> <
> #7706 <https://github.com/girlscript/winter-of-contributing/pull/7706>>
> to @Ruchika-20 <https://github.com/Ruchika-20>
> https://github.com/Ruchika-20. — You are receiving this because you were
> assigned. Reply to this email directly, view it on GitHub <#7706 (comment)
> <https://github.com/girlscript/winter-of-contributing/pull/7706#event-5598417066>>,
> or unsubscribe
> https://github.com/notifications/unsubscribe-auth/AUEIB54KAFJ4XHEOEHJGOITULKNCLANCNFSM5HYINP4A
> . Triage notifications on the go with GitHub Mobile for iOS
> https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675
> or Android
> https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub
> .
>
> @Ruchika-20 <https://github.com/Ruchika-20>
>
> actually i did not understood the nomenclature of url and asked about the
> same to isha but she was not able to see my msg so i renamed according to
> mine understanding.....Can you please correct me so that i can make the
> necessary changes? Thank you.
> … <#m_5261266020893853437_>
> On Wed, Nov 10, 2021 at 10:02 PM Abhishek Sharma *@*.***> wrote: Assigned
> #7706 <https://github.com/girlscript/winter-of-contributing/pull/7706> <
> #7706 <https://github.com/girlscript/winter-of-contributing/pull/7706>>
> to @Ruchika-20 <https://github.com/Ruchika-20>
> https://github.com/Ruchika-20. — You are receiving this because you were
> assigned. Reply to this email directly, view it on GitHub <#7706 (comment)
> <https://github.com/girlscript/winter-of-contributing/pull/7706#event-5598417066>>,
> or unsubscribe
> https://github.com/notifications/unsubscribe-auth/AUEIB54KAFJ4XHEOEHJGOITULKNCLANCNFSM5HYINP4A
> . Triage notifications on the go with GitHub Mobile for iOS
> https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675
> or Android
> https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub
> .
>
> @Ruchika-20 <https://github.com/Ruchika-20> Sorry I was busy,
> You have to make a markdown file and follow this template "
> https://github.com/girlscript/winter-of-contributing/blob/Machine_Learning/Machine_Learning/Demo/ML_%23_%23_Audio%20Template_(A).md
> "
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/girlscript/winter-of-contributing/pull/7706#issuecomment-966259759>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUEIB533ZUD3CLV3ZZMGBCDULOYTNANCNFSM5HYINP4A>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>

Isha307 commented:
> [![Your_Title](Image_link_for_Audio_thumbnail)](Your_audio_link "Your_title") please explain this format
> […](#)
> On Thu, Nov 11, 2021 at 5:53 PM isha ***@***.***> wrote: actually i did not understood the nomenclature of url and asked about the same to isha but she was not able to see my msg so i renamed according to mine understanding.....Can you please correct me so that i can make the necessary changes? Thank you. … <#m_5261266020893853437_> On Wed, Nov 10, 2021 at 10:02 PM Abhishek Sharma *@*.***> wrote: Assigned #7706 <#7706> < #7706 <#7706>> to @Ruchika-20 <https://github.com/Ruchika-20> https://github.com/Ruchika-20. — You are receiving this because you were assigned. Reply to this email directly, view it on GitHub <#7706 (comment) <[#7706 (comment)](https://github.com/girlscript/winter-of-contributing/pull/7706#event-5598417066)>>, or unsubscribe https://github.com/notifications/unsubscribe-auth/AUEIB54KAFJ4XHEOEHJGOITULKNCLANCNFSM5HYINP4A . Triage notifications on the go with GitHub Mobile for iOS https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675 or Android https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub . @Ruchika-20 <https://github.com/Ruchika-20> actually i did not understood the nomenclature of url and asked about the same to isha but she was not able to see my msg so i renamed according to mine understanding.....Can you please correct me so that i can make the necessary changes? Thank you. … <#m_5261266020893853437_> On Wed, Nov 10, 2021 at 10:02 PM Abhishek Sharma *@*.***> wrote: Assigned #7706 <#7706> < #7706 <#7706>> to @Ruchika-20 <https://github.com/Ruchika-20> https://github.com/Ruchika-20. — You are receiving this because you were assigned. Reply to this email directly, view it on GitHub <#7706 (comment) <[#7706 (comment)](https://github.com/girlscript/winter-of-contributing/pull/7706#event-5598417066)>>, or unsubscribe https://github.com/notifications/unsubscribe-auth/AUEIB54KAFJ4XHEOEHJGOITULKNCLANCNFSM5HYINP4A . Triage notifications on the go with GitHub Mobile for iOS https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675 or Android https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub . @Ruchika-20 <https://github.com/Ruchika-20> Sorry I was busy, You have to make a markdown file and follow this template " https://github.com/girlscript/winter-of-contributing/blob/Machine_Learning/Machine_Learning/Demo/ML_%23_%23_Audio%20Template_(A).md " — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub <[#7706 (comment)](https://github.com/girlscript/winter-of-contributing/pull/7706#issuecomment-966259759)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AUEIB533ZUD3CLV3ZZMGBCDULOYTNANCNFSM5HYINP4A> . Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

Your_Title - Name of the topic
Image_link_for_Audio_thumbnail - Link of the image you want to keep as thumbnail
Your_audio_link - Google drive link

abhisheks008 commented:
@Ruchika-20 Read the Demo folder's Readme.md file, everything is clearly written there.

Ruchika-20 commented:
what should be my thumbnail?


On Fri, Nov 12, 2021 at 9:48 AM Abhishek Sharma ***@***.***>
wrote:

> @Ruchika-20 <https://github.com/Ruchika-20> Read the Demo folder's
> Readme.md file, everything is clearly written there.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/girlscript/winter-of-contributing/pull/7706#issuecomment-966808131>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUEIB57JV45CB4RK37TNCLLULSISPANCNFSM5HYINP4A>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>

Ruchika-20 commented:
ok

On Fri, Nov 12, 2021 at 9:48 AM Abhishek Sharma ***@***.***>
wrote:

> @Ruchika-20 <https://github.com/Ruchika-20> Read the Demo folder's
> Readme.md file, everything is clearly written there.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/girlscript/winter-of-contributing/pull/7706#issuecomment-966808131>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUEIB57JV45CB4RK37TNCLLULSISPANCNFSM5HYINP4A>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>

Ruchika-20 commented:
where is the asset folder?

On Fri, Nov 12, 2021 at 11:14 AM Ruchika Singh ***@***.***>
wrote:

> ok
>
> On Fri, Nov 12, 2021 at 9:48 AM Abhishek Sharma ***@***.***>
> wrote:
>
>> @Ruchika-20 <https://github.com/Ruchika-20> Read the Demo folder's
>> Readme.md file, everything is clearly written there.
>>
>> —
>> You are receiving this because you were mentioned.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/girlscript/winter-of-contributing/pull/7706#issuecomment-966808131>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AUEIB57JV45CB4RK37TNCLLULSISPANCNFSM5HYINP4A>
>> .
>> Triage notifications on the go with GitHub Mobile for iOS
>> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
>> or Android
>> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>>
>>
>

abhisheks008 commented:
@Ruchika-20 

1. When you are surfing on YouTube, did you get the URLs of the videos directly? or, you just see that thumbnail and click on that. Here also you are providing the thumbnail. Secondly, what will be your thumbnail, that's up to you because you are the creator here and you know how to represent your content!
2. Every chapter/subdirectory has it's assets folder to store the thumbnails, dataset. You'll find them in each and every directory.

Ruchika-20 commented:
Changes done @abhisheks008  . Kindly have a look

Isha307 reviewed (changes requested):
@Ruchika-20 you added the link in .ipynb file.
For audio and video, you have to write the content into markdown file and also the link is not working.

Ruchika-20 commented:
Okay I will correct it till EOD

On Sat, Nov 13, 2021, 9:02 AM isha ***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
>
> @Ruchika-20 <https://github.com/Ruchika-20> you added the link in .ipynb
> file.
> For audio and video, you have to write the content into markdown file and
> also the link is not working.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/girlscript/winter-of-contributing/pull/7706#pullrequestreview-805334091>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUEIB52DRLEN3FL2XUJ3G3TULXL4XANCNFSM5HYINP4A>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>

abhisheks008 commented:
@Ruchika-20 Do the needful changes.

Ruchika-20 commented:
i have externals from monday so i am not getting time but i will surely do
it by today

On Sun, Nov 14, 2021 at 9:30 AM Abhishek Sharma ***@***.***>
wrote:

> @Ruchika-20 <https://github.com/Ruchika-20> Do the needful changes.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/girlscript/winter-of-contributing/pull/7706#issuecomment-968201692>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUEIB5Z6KTHH37YMV57ZMWDUL4X7VANCNFSM5HYINP4A>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>

Ruchika-20 commented:
changes done...have a look

On Sun, Nov 14, 2021 at 9:30 AM Abhishek Sharma ***@***.***>
wrote:

> @Ruchika-20 <https://github.com/Ruchika-20> Do the needful changes.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/girlscript/winter-of-contributing/pull/7706#issuecomment-968201692>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUEIB5Z6KTHH37YMV57ZMWDUL4X7VANCNFSM5HYINP4A>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>

abhisheks008 commented:
Good work @Ruchika-20 🎉 
All the best for your externals.

Ruchika-20 commented:
Thank You :)

On Sun, Nov 14, 2021 at 6:57 PM Abhishek Sharma ***@***.***>
wrote:

> Good work @Ruchika-20 <https://github.com/Ruchika-20> 🎉
> All the best for your externals.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/girlscript/winter-of-contributing/pull/7706#issuecomment-968290991>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUEIB5345KYD7UIKSFODKFLUL62MRANCNFSM5HYINP4A>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>
